core/trino-main/src/main/java/io/trino/memory/LocalMemoryManagerExporter.java

```diff
@@ -21,49 +21,39 @@
 import javax.annotation.concurrent.GuardedBy;
 import javax.inject.Inject;
 
-import java.util.ArrayList;
-import java.util.List;
-
 import static java.util.Objects.requireNonNull;
 
 public final class LocalMemoryManagerExporter
 {
+    public static final String EXPORTED_POOL_NAME = "general";
     private final MBeanExporter exporter;
     @GuardedBy("this")
-    private final List<MemoryPool> pools = new ArrayList<>();
+    private MemoryPool pool;
```

Comment:
`final`. 
still `@GuardedBy("this")`?

Reply:
I agree. Also it feels we did not need that synchronization before neither (give destroy is called once)

Reply:
Actually we do not need pool field at all - just a boolean flag if we exported JMX bean.